apps/audits/serializers.py

```diff
@@ -34,6 +34,21 @@ class Meta:
         }
 
 
+class JobsAuditSerializer(JobSerializer):
+    class Meta(JobSerializer.Meta):
+        fields = JobSerializer.Meta.fields
+
+    def validate(self, attrs):
+        allowed_fields = {'is_periodic'}
+        submitted_fields = set(attrs.keys())
+        invalid_fields = submitted_fields - allowed_fields
+        if invalid_fields:
+            raise serializers.ValidationError(
+                f"Updating  {', '.join(invalid_fields)} fields is not allowed"
+            )
+        return attrs
+
+
 class FTPLogSerializer(serializers.ModelSerializer):
     operate = LabeledChoiceField(choices=OperateChoices.choices, label=_("Operate"))
 
```

Comment:
Based on your request, no major code changes were mentioned for comparison. The existing structure of JobExecutionSerializer and JobSerializer appears to be similar to what we already discussed earlier:

```python
# ... Code here ...
@shared_task
def create_job_execution(*args, session_id=None, job_name='default', user_id=None):
    if 'session' in locals():
        session: Session = session

    # Handle cases where no job execution needs to occur based on the provided variables...
```

There were some minor adjustments made to `Operations`. These appear to be unrelated to the main function but could possibly have implications when running these operations.

For reference, you can find all my responses so far as well as updated knowledge cutoff information at <https://docs.openai.com/faq/cutoff>